src/Umbraco.Web.UI.Client/src/views/packager/views/install-local.controller.js

```diff
@@ -95,16 +95,11 @@
         }
 
         function installPackage() {
-            vm.installState.status = "Installing";
+            vm.installState.status = "Importing";
 
             //TODO: If any of these fail, will they keep calling the next one?
```

Comment:
This TODO is done from what I understand with the `$q.reject` taking place in the `installError()` function